src/main/java/io/fabric8/jenkins/openshiftsync/BuildWatcher.java

```diff
@@ -363,22 +364,21 @@ private static synchronized void clearNoBCList() {
     // trigger any builds whose watch events arrived before the
     // corresponding build config watch events
     public static synchronized void flushBuildsWithNoBCList() {
-        HashSet<Build> clone = (HashSet<Build>) buildsWithNoBCList.clone();
-        clearNoBCList();
```

Comment:
There were no other references to `clearNoBCList()` ... go ahead and delete that as well